docs/Homebrew-homebrew-core-Merge-Checklist.md

```diff
@@ -1,22 +1,135 @@
-# Homebrew/homebrew-core Merge Checklist
+# Homebrew/homebrew-core Maintainer Guide
 
-The following checklist is intended to help maintainers decide on
-whether to merge, request changes or close a PR. It also brings more
-transparency for contributors in addition to
-[Acceptable Formulae](Acceptable-Formulae.md) requirements.
+## Quick merge checklist
+
+A detailed checklist can be found [below](#detailed-merge-checklist). This is all that really matters:
+
+- Ensure the name seems reasonable.
+- Add aliases.
+- Ensure it uses `keg_only :provided_by_macos` if it already comes with macOS.
+- Ensure it is not a library that can be installed with
+  [gem](https://en.wikipedia.org/wiki/RubyGems),
+  [cpan](https://en.wikipedia.org/wiki/Cpan) or
+  [pip](https://pip.pypa.io/en/stable/).
```

Comment:
Do we actually check for this?

Reply:
HB ships at least a few `pip`-installable tools. The most well-known is probably `youtube-dl`.

But even the ones I know of which are libraries provide command-line interfaces, so it depends on the interpretation of this section.

I’d be for continuing to provide those. HB’s usefulness grows if a user doesn’t have to learn/use/update a language’s package manager to keep a single popular tool up-to-date.

Reply:
Yeh, hence the "library" bit

Reply:
We've recently started shipping `six`, which is a pip installable library to make packaging easier. I don't think this is still actually enforced (and I don't see any good reasons why we would)

Reply:
If we need it for our own benefit: cool/fine. The intent is that we don't have a million Python libraries and duplicate that package manager (like e.g. Debian does)

Reply:
I thought `apt` was your dream package manager 😄